src/test_l1_dsAudio.c

```diff
@@ -77,9 +77,10 @@
 #include <ut_log.h>
 #include <ut_kvp_profile.h>
 
-#define NUM_OF_PORTS (sizeof(kPorts) / sizeof(kPorts[0]))
+#define NUM_OF_PORTS 4
```

Comment:
We can't say no of ports will be 4 for all platforms, it should be from profile

Reply:
yes , currently we cannot add the variable name to the array size , it will give a compilation error.